pyproject.toml

```diff
@@ -57,10 +57,10 @@ dependencies = [
     "joblib>=1",
     "matplotlib>=3.8",
     "monty>=2024.7.29",
-    "networkx>=3",  # PR #4116
+    "networkx>=3",  # PR4116
     "palettable>=3.3.3",
     "pandas>=2",
-    "plotly>=4.5.0",
+    "plotly>=5.0.0",  # PR4138
```

Comment:
Try to fix (minimal version not tested yet, https://pypi.org/project/plotly/#history):

```
../../../micromamba/envs/pmg/lib/python3.12/site-packages/plotly/graph_objs/__init__.py:94045
  /home/runner/micromamba/envs/pmg/lib/python3.12/site-packages/plotly/graph_objs/__init__.py:94045: SyntaxWarning:
  
  invalid escape sequence '\d'

../../../micromamba/envs/pmg/lib/python3.12/site-packages/plotly/basedatatypes.py:43
  /home/runner/micromamba/envs/pmg/lib/python3.12/site-packages/plotly/basedatatypes.py:43: SyntaxWarning:
  
  invalid escape sequence '\['

../../../micromamba/envs/pmg/lib/python3.12/site-packages/_plotly_utils/utils.py:214
../../../micromamba/envs/pmg/lib/python3.12/site-packages/_plotly_utils/utils.py:214
  /home/runner/micromamba/envs/pmg/lib/python3.12/site-packages/_plotly_utils/utils.py:214: SyntaxWarning:
  
  "is" with 'str' literal. Did you mean "=="?
```